debugstart.ts

```diff
@@ -6,15 +6,16 @@ import { importObject, addLibs  } from "./tests/import-object.test";
 // entry point for debugging
 async function debug() {
   var source = `
-class C(object):
-  def f(self: C) -> int:
-    if True:
-      return 0
-    else:
-      return`
+  set_1 : set[int] = None
+    set_1 = {1,2}
+    set_1.add(3)
+    print(3 in set_1)`
+
   const ast = parse(source);
   
   const repl = new BasicREPL(await addLibs());
+  //const result = repl.tc(source);
```

Comment:
Don't include your debugging into the PR.

Reply:
Got it!